dusty/systems/virtualbox/__init__.py

```diff
@@ -115,6 +115,7 @@ def _stop_docker_vm():
     """Stop the Dusty VM if it is not already stopped."""
     check_call_demoted(['docker-machine', 'stop', constants.VM_MACHINE_NAME], redirect_stderr=True)
 
+@memoized
 def _get_vm_config():
     return check_output_demoted(['VBoxManage', 'showvminfo', '--machinereadable', constants.VM_MACHINE_NAME]).splitlines()
```

Comment:
why is this worth memoizing?  Is it really slow?  seems like needless speedup with costs.